components/Content.tsx

```diff
@@ -27,9 +27,9 @@ const data = {
     network: Network,
     token: Token,
     tokenEndpoint: TokenEndpoint,
-    securityTokens1: SecurityTokens1,
-    securityTokens2: SecurityTokens2,
-    firewallAndPorts: FirewallAndPorts,
+    securityAndTokens1: SecurityTokens1,
```

Comment:
Can you change these back to securityTokensx @Nixon-Samuel?
The component name needs to be the same as the file name - the 'and' is extra here, and the Algolia script won't be able to replace the content

Reply:
done